01/app.js

```diff
@@ -1 +1,9 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const element = document.getElementsByClassName('comments__item' && 'comments__item--newest');
```

Comment:
Ten zapis jest nieprawidłowy, chociaż w tym przypadku "działa".
Samo `'comments__item' && 'comments__item--newest'` daje nam `comments__item--newest` i dlatego to działa.
Prawidłowo by było `'comments__item comments__item--newest`